Jenkinsfile

```diff
@@ -189,7 +189,7 @@ pipeline {
   parameters {
     booleanParam(defaultValue: false, name: 'e2e_continuous')
     booleanParam(defaultValue: false, name: 'run_as_nightly')
-    booleanParam(defaultValue: true, name: 'run_e2e_test')
+    booleanParam(defaultValue: false, name: 'run_e2e_test')
```

Comment:
can this variable not be overridden from the jenkins UI?

Reply:
only when run manually IMHO (i.e. "build with parameters") - bors/scheduler runs will always use the defaults; there are also global environment variables - a bit less user friendly re UI however default could be changed without touching the code

Reply:
Yes it can. But we want to disable it indefinitely in the Mayastor builds. E2e tests will still be run, but in another pipeline.